references/classification/train.py

```diff
@@ -230,7 +230,7 @@ def main(args):
     criterion = nn.CrossEntropyLoss(label_smoothing=args.label_smoothing)
 
     if args.norm_weight_decay is None:
-        parameters = model.parameters()
+        parameters = [p for p in model.parameters() if p.requires_grad]
```

Comment:
Nit fix to align reference scripts. This is not strictly needed in Classification since we don't freeze weights but that's more generic.